lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -347,6 +358,7 @@ def _connect_uncached(self):
                 password=self._play_context.password,
                 timeout=self._play_context.timeout,
                 port=port,
+                banner_timeout=self.get_option('banner_timeout'),
```

Comment:
This feature was added in paramiko 1.15.0: https://www.paramiko.org/changelog.html#1.15.0

That's why the tests are failing on the centos6 test container currently.

This should only be set if paramiko supports it. The documentation should also note the paramiko version requirement. I'd also include a warning if it's set by the user and can't be applied.

Reply:
Thanks for reviewing it. Fix the comments.